pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -174,12 +631,11 @@ func (r *Builder) Tasks(vmRef ref.Ref) (list []*plan.Task, err error) {
 			vmRef.String())
 	}
 
-	for _, va := range vm.AttachedVolumes {
+	for _, va := range vm.Volumes {
 		gb := int64(va.Size)
 		list = append(
 			list,
 			&plan.Task{
-				Name: fmt.Sprintf("%s-%s", r.Migration.Name, va.ID),
```

Comment:
why?

Reply:
Mmm, that line was removed by accident I am afraid... 😞